src/main/java/xyz/groundx/caver_ext_kas/CaverExtKAS.java

```diff
@@ -388,6 +388,7 @@ public void initWalletAPI(int chainId, String accessKeyId, String secretAccessKe
     public void initWalletAPI(String chainId, String accessKeyId, String secretAccessKey, String url) {
         kas.initWalletAPI(chainId, accessKeyId, secretAccessKey, url);
         kas.wallet.setRPC(this.rpc);
+        kas.wallet.setEndpointUrl(url);
```

Comment:
I think it is incorrect code. Did you test this code?

Reply:
Sorry for this. I fixed this logic.